### ISSUE TITLE
A worldsNoTag= option in the main config.

### ISSUE THREAD

**Issue by VladQc** [drtshock/Factions#1412]
### Idea
There should be a worldsNoTag= option just like there is a worldsNoClaiming= option. Very useful for customization and for servers with multiple gamemodes.

For exemple, in my server, I have PVP faction and skyblock. When a players types in chat while being in the skyblock world the chat looks like that:

*****MyFactionName [SkyBlock] VladQc: I like apples.**

I just think it would look cleaner like this:

**[SkyBlock] VladQc: I like apples.**

### Details

I know we can use essential chat or other chat plugins to do this but I've never seen an up to date plugin that does what I want.
I tried to use Luck Perm with placeholders or whatever but the documentation about it is confusing and it doesnt works. I also tried to put **Faction."*" to false** if a player was in the Skyblock world but the Tag is still there even if the Player has no faction permissions at all.

It could look like this:

worldsNoTag=[
            "bskyblock_world",
        ]

It would solve many problems.

**Edit: After giving it some thought, I think that maybe a permission node could do all of that. A permision node that you have by default called Faction.ShowTagInChat or something similar.**

**Comment by brokuka** [comment]
Yep, there is bug if you using essentials chat and disabled faction in specific world. In chat just [FACTION], not hiding. Maybe there is any solutions ?
